lib/spack/docs/configuration.rst

```diff
@@ -408,6 +408,7 @@ Spack understands several special variables. These are:
   <https://docs.python.org/2/library/tempfile.html#tempfile.tempdir>`_
   variable.
 * ``$user``: name of the current user
+* ``$user_config``: name of the current user
```

Comment:
The description seems a cut and paste typo

Reply:
Odd, I must have had a stale copy in my stage, thanks for catching that!